net.worldofpadman.WoP.yaml

```diff
@@ -35,8 +35,11 @@ modules:
         sha256: 526d5936225b37fa8a5b94a7c02dc600126d1f2fb1155fd0e372c36075344335
```

Comment:
```
url: http://www.zock-server.de/downloads/WoP_1.6.2/wop-1.6.2-full-unified.zip
```
This also needs to be fixed. There is no WoP 1.7.0 archive available on the zock-server.de server yet.

Reply:
Wow, I did not realize that it is available from there. Thanks!